test/track.ts

```diff
@@ -132,6 +132,18 @@ test('#trackAnonymous works', (t) => {
   );
 });
 
+test('#trackAnonymous ignores blank anonymousId', (t) => {
+  sinon.stub(t.context.client.request, 'post');
+  t.context.client.trackAnonymous('', { name: 'purchase', data: 'yep' })
+  t.context.client.trackAnonymous('123', { name: 'purchase', data: 'yep' });
+  t.truthy(
```

Comment:
This might be a false positive because this doesn't check that the stub was called twice. I would either:
- Add a `t.truthy` after the initial `trackAnonymous` block to check that it was called without an `anonymous_id`
  - This would also mean you should add a check in the second `t.truthy` block to see that `anonymous_id` *was* added
- Add a `t.truthy` call to check the stub's call count: `(t.context.client.request as SinonStub).callCount`